cmd/cosign/cli/env.go

```diff
@@ -21,29 +21,41 @@ import (
 	"strings"
 
 	"github.com/spf13/cobra"
+
+	"github.com/sigstore/cosign/cmd/cosign/cli/options"
+	"github.com/sigstore/cosign/pkg/cosign/env"
 )
 
 func Env() *cobra.Command {
-	return &cobra.Command{
+	o := &options.EnvOptions{}
+
+	cmd := &cobra.Command{
 		Use:   "env",
 		Short: "Prints Cosign environment variables",
 		Args:  cobra.NoArgs,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			for _, e := range getEnv() {
+			// Print COSIGN_ environment variables
+			env.PrintEnv(o.ShowDescriptions, o.ShowSensitiveValues)
+
+			// Print external environment variables (SIGSTORE_ and TUF_)
```

Comment:
Maybe worth printing a comment about these (`# These are environment variables that Cosign doesn't specifically know about, but with prefixes that look like they might be used by some of Cosign's dependencies` or something)

Reply:
(if `o.ShowDescriptions` is set)

Reply:
That's a good point, I added something similar.